### DISCUSSION TITLE
FPS Land as demo

### DISCUSSION THREAD

**Discussion by Luzifix** [FirstGearGames/FishNet#340] [Feature Request]
Greetings,

At present, I am contemplating the idea of sponsoring Fishnet and seeking a comparison between its Free and Pro versions. To facilitate this, I suggest creating a compiled demo of FPS Land, which can be downloaded and tested by all. Such a demo would provide an effective means of comparing the two integrations, enabling users to test features such as lag compensation and full server authority.

**Comment by FirstGearGames** [comment]
I could do this but will need to pack the executable so it's more difficult to reverse engineer.

**Comment by FirstGearGames** [comment]
I've decided against this for now. There's no timeline I can provide for when this will happen and it's a bit of extra work on-top of our already busy schedule. Sorry.
